exasol/analytics/schema/statements.py

```diff
@@ -105,7 +111,13 @@ def col_val(name: str) -> str:
             return str(val)
 
         names = sorted(values)
-        self._columns += [self._lookup_column(n) for n in names]
+        additional = [self._lookup_column(n) for n in names]
+        if duplicates := set(additional) & set(self._columns):
+            n = len(duplicates)
+            message = f"{n} duplicate columns:" if n > 1 else "duplicate column"
```

Comment:
```suggestion
            message = f"{n} duplicate columns:" if n > 1 else "duplicate column:"
```

Reply:
Actually, I wanted to create the strings as [demonstrated in the test](https://github.com/exasol/advanced-analytics-framework/pull/269/files#diff-a04463c7bd7e9828ebf1fd03e5bb359f1af6a90ddd85c68a914db8018551afd7R27):

* Can't add duplicate column "A".
* Can't add 2 duplicate columns: "A", "B".

Reply:
See latest push.